monitoring/local_monitoring/bulk_demo/Bulk_demo.md

```diff
@@ -10,14 +10,13 @@ recommendations. It also has settings to generate recommendations at both the co
 Bulk returns a `job_id` as a response to track the job status. The user can use the `job_id` to monitor the
```

Comment:
Its a good idea to move this to a separate dir ... however please rename this as README.md and also add a comment in the local_monitoring README about this demo as well

Reply:
Done